### DISCUSSION TITLE
New Source: Teamtailor

### DISCUSSION THREAD

**Discussion by Pwaldi** [airbytehq/airbyte#35216] [New Connector Request]
## Tell us about the new connector you’d like to have
* Which source and which destination?
> We need to get data on job applications etc. from Teamtailor
* Do you need a specific version of the underlying data source e.g: you specifically need support for an older version of the API or DB? 
> The latest version of the Teamtailor API should be fine

## Describe the context around this new connector
* Why do you need this integration? How does your team intend to use the data? This helps us understand the use case.
> We need to get data on job openings, applications etc. to analyze our full hiring funnel
* How often do you want to run syncs?
> on a daily basis
* If this is an API source connector, which entities/endpoints do you need supported?
> For our use case we would mainly need `https://api.teamtailor.com/v1/jobs` and `https://api.teamtailor.com/v1/job-applications`
* If the connector is for a paid service, can we name you as a mutual user when we subscribe for an account? Which company should we name? 
> Yes, Neue Fische GmbH

## Describe the alternative you are considering or using
What are you considering doing if you don’t have this integration through Airbyte?
> Using a Python Script that loads the data from the two specific endpoints

## Are you willing to submit a PR?
> We would be happy to try out developing the integration on our own but are of course happy for any help available.

**Comment by Pwaldi** [comment]
I would jump on developing this

**Comment by sajarin** [comment]
Hey @Pwaldi thanks for the issue and for jumping on the development yourself. Looking forward to your contribution! Let us know if you have any questions.

[a comment by an automated account is not shown]
